src/app/dim-ui/PressTip.m.scss

```diff
@@ -13,18 +11,14 @@ $horizontal-padding: 11px;
 }
 
 .tooltip {
-  --tooltip-ribbon-color: #8e8e9e;
-  --tooltip-background-color: #151523;
```

Comment:
This variable gets used by [the `tooltip-section-color` mixin](https://github.com/DestinyItemManager/DIM/blob/master/src/app/dim-ui/_tooltip-mixins.scss#L22) for the tooltip section separator. Note that the tooltip's background colour [can be overridden](https://github.com/DestinyItemManager/DIM/blob/4be7854f229d76108e3ebbff41545bc6f9abeaf4/src/app/shell/RefreshButton.m.scss#L36), so this probably still needs to be somewhat dynamic.

![dim-tooltip-section-border-colour](https://github.com/DestinyItemManager/DIM/assets/17512262/9d429565-5e8c-4021-886b-c9df0538b7f3)

Reply:
We actually used to have tinted separators at one point, but [ended up moving away from them](https://github.com/DestinyItemManager/DIM/pull/8638) after receiving feedback that they distracted from the text.

I don't think the separator is necessary for coloured sections though, so it might be easier to override `border-top-color` to `transparent` in the `tooltip-section-color` mixin rather than try pull the tooltip background colour from somewhere.